java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -316,4 +319,47 @@ public interface StringBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> append(String key, String value);
+
+    /**
+     * Returns the longest common subsequence between strings stored at <code>key1</code> and <code>
+     * key2</code>.
+     *
+     * @since Redis 7.0 and above.
+     * @apiNote When in cluster mode, <code>key1</code> and <code>key2</code> must map to the same
+     *     hash slot.
+     * @see <a href="https://valkey.io/commands/lcs/">valkey.io</a> for details.
+     * @param key1 The key that stores the first string.
+     * @param key2 The key that stores the second string.
+     * @return A <code>String</code> containing the longest common subsequence between the 2 strings.
+     *     An empty <code>String</code> is returned if the keys do not exist or have no common
+     *     subsequences.
+     * @example
+     *     <pre>{@code
+     * client.set("testKey1", "abcd");
+     * client.set("testKey2", "axcd");
+     * assertEquals("acd", client.lcs("testKey1", "testKey2").get());
```

Comment:
`assertEquals` and other assert functions are designed for tests.
```suggestion
     * String result = client.lcs("testKey1", "testKey2").get();
     * assert result.equals("acd");
```